compute_size_ratio: 1
integration_rate: 1
rendering_rate: 1
tracking_rate: 1
volume_resolution: [256,256,256]
#volume_resolution: [512,512,512]
volume_direction: [4.0,4.0,4.0]
volume_size: [8,8,8]
pyramid: [10,5,4]
mu: 0.2
icp_threshold: 1e-03
